src/tests/ftest/io/eight_servers.py

```diff
@@ -21,96 +21,41 @@
   Any reproduction of computer software, computer software documentation, or
   portions thereof marked with this legend must also reproduce the markings.
 '''
-from __future__ import print_function
+from ior_single_server import IorTestBase
```

Comment:
As commented below, I think this needs to come from a utility module, not a test module.

Reply:
Agreed, I will implement the change